experimental/apps-mcp/cmd/mcp.go

```diff
@@ -0,0 +1,91 @@
+package mcp
+
+import (
+	"github.com/databricks/cli/cmd/root"
+	mcplib "github.com/databricks/cli/experimental/apps-mcp/lib"
+	"github.com/databricks/cli/experimental/apps-mcp/lib/server"
+	"github.com/databricks/cli/libs/cmdctx"
+	"github.com/databricks/cli/libs/log"
+	"github.com/spf13/cobra"
+)
+
+func NewMcpCmd() *cobra.Command {
+	var warehouseID string
+	var allowDeployment bool
+	var withWorkspaceTools bool
+	var dockerImage string
+	var useDagger bool
+
+	cmd := &cobra.Command{
+		Use:   "apps-mcp",
```

Comment:
Blocking: this should be called apps_mcp.go and must be marked as hidden.

Reply:
done